src/rsz/test/repair_fanout7.ok

```diff
@@ -12,9 +12,8 @@ load DFF_X1 CK D 300000 metal3 1000
 [INFO ODB-0133]     Created 155 nets and 459 connections.
 worst slack -23.63
 [INFO RSZ-0094] Found 150 endpoints with setup violations.
-[INFO RSZ-0045] Inserted 78 buffers, 1 to split loads.
-[INFO RSZ-0041] Resized 527 instances.
-[INFO RSZ-0049] Cloned 7 instances.
+[INFO RSZ-0045] Inserted 40 buffers, 1 to split loads.
+[INFO RSZ-0041] Resized 364 instances.
+[INFO RSZ-0049] Cloned 4 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
-Repair timing output passed/skipped equivalence test
-worst slack -3.75
+worst slack -2.84
```

Comment:
I thought this is off by default - why is there a diff?

Reply:
What is "off my default"

Reply:
The diff comes from some cleanup that happened during the implementation of the infrastructure to redo the "undo" operation. The results are better.

Reply:
typo : off by default

Reply:
Does "The results are better." imply you ran a secure CI?

Reply:
No. All it implies is that these particular results are better (-3.75 vs. -2.84). If you think there is a need to run the secure CI for this change then I can try to remember how to do that. LMK.